### PR TITLE
980: updated document categories 

### PR DESCRIPTION
**Pull request by rachaelparris**

### CODE REVIEW

**Review by kkoskelin — APPROVED** [review]

File: shared/src/business/entities/externalDocument/ExternalDocumentNonStandardA.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ ExternalDocumentNonStandardA.errorToMessageMap = {
 
 ExternalDocumentNonStandardA.schema = joi.object().keys({
   category: joi.string().required(),
+  documentTitle: joi.string().optional(),
```
Comment (wbyoko) [inline shared/src/business/entities/externalDocument/ExternalDocumentNonStandardA.js:28]: should `documentTitle` be `required`?

Comment (rachaelparris) [reply shared/src/business/entities/externalDocument/ExternalDocumentNonStandardA.js:28]: I didn't want the user to get an error message about it. Is there a way to make it required, but not return an error message about that field to the user? Seems like maybe we've done that in another place, but I couldn't find it.

**Review by ericsorenson — APPROVED** [review]
🍕

**Pull request merged**